x-pack/plugins/reporting/server/routes/diagnostic/config.ts

```diff
@@ -16,6 +16,14 @@ import { authorizedUserPreRoutingFactory } from '../lib/authorized_user_pre_rout
 const KIBANA_MAX_SIZE_BYTES_PATH = 'csv.maxSizeBytes';
 const ES_MAX_SIZE_BYTES_PATH = 'http.max_content_length';
 
+const numberToByteSizeValue = (value: number | ByteSizeValue) => {
+  if (typeof value === 'number') {
+    return new ByteSizeValue(value);
```

Comment:
Unfortunately, this construction makes this function not possible to be in `common`

Reply:
Gotcha, thanks for checking